### PR TITLE
[gem] Fix yanked mimemagic

### PR DESCRIPTION
**Pull request by hallelujah**
The author decided to yank all versions but 0.4.0 and 0.3.6 of mimemagic

I do not know the reason yet but we should ensure that our project can be installed

Credits: @raelga

### CODE REVIEW

**Review by HemaHG — APPROVED** [review]

**Review by josemigallas — DISMISSED** [review]

**Review by HemaHG — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -27,6 +27,14 @@ GIT
     message_bus_client (0.2.0)
       excon (~> 0.45)
 
+GIT
+  remote: https://github.com/3scale/mimemagic
+  revision: 696db81993d60812b70c2bff2c226cba4670acd9
+  branch: 0.3-3scale
```
Comment (guicassolato) [inline Gemfile.lock:33]: So it's nokogiri v1.10.10.

Comment (hallelujah) [reply Gemfile.lock:33]: Yes it is what we have in our Gemfile

**Review by guicassolato — DISMISSED** [review]
We still have some diffs showing up when we `bundle exec rake ci:license_finder:run` locally. Needs investigation.

**Review by guicassolato — APPROVED** [review]
When running license_finder on macOs, only "unknown" licenses now seem to be these couple ones:

- https://rubygems.org/gems/prawn-format/versions/0.2.1
- https://github.com/component/emitter/tree/1.1.1 (license file added in v1.1.3)

```
$ ag unknown doc/licenses/licenses.xml -B4
4341-        <packageName>emitter-component</packageName>
4342-        <version>1.1.1</version>
4343-        <licenses>
4344-                      <license>
4345:              <name>UNKNOWN</name>
--
9503-        <packageName>prawn-format</packageName>
9504-        <version>0.2.1</version>
9505-        <licenses>
9506-                      <license>
9507:              <name>unknown</name>
--
9513-        <packageName>prawn-layout</packageName>
9514-        <version>0.2.1</version>
9515-        <licenses>
9516-                      <license>
9517:              <name>unknown</name> 
```

**Pull request merged**